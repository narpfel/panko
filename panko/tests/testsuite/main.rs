#![feature(unqualified_local_imports)]

fn main() {
    panko_compiletest::run()
}
