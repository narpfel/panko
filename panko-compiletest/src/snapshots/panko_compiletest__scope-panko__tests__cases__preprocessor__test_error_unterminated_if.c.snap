---
source: panko/tests/testsuite/main.rs
info:
  program: panko
  args:
    - "--print=scopes"
    - "--stop-after=scopes"
    - panko/tests/cases/preprocessor/test_error_unterminated_if.c
---
success: false
exit_code: 1
----- stdout -----

----- stderr -----
Error: unterminated `#if` directive
   ╭─[panko/tests/cases/preprocessor/test_error_unterminated_if.c:1:1]
   │
 1 │ #if true
   │ ─┬─  
   │  ╰─── this directive does not have a corresponding `#endif`
───╯

Error: unterminated `#ifdef` directive
   ╭─[panko/tests/cases/preprocessor/test_error_unterminated_if.c:2:1]
   │
 2 │ #ifdef __LINE__
   │ ───┬──  
   │    ╰──── this directive does not have a corresponding `#endif`
───╯
